.github/workflows/publish.yml

```diff
@@ -29,13 +29,13 @@ jobs:
       run: |
         cd ./stubs && python3 setup.py sdist bdist_wheel --universal && cd -
     - name: Publish metaflow-stubs package
-      uses: pypa/gh-action-pypi-publish@9b8e7336db3f96a2939a3e9fa827c62f466ca60d # master
+      uses: pypa/gh-action-pypi-publish@release/v1.8
       with:
         user: __token__
         password: ${{ secrets.pypi_password }}
-        packages-dir: ./stubs/dist
```

Comment:
the doc does say `packages-dir`. I think it was "fixed": https://github.com/pypa/gh-action-pypi-publish/blob/c12cc61414480c03e10ea76e2a0a1a17d6c764e2/action.yml#L28

Reply:
reverted...